vms/vms/settings.py

```diff
@@ -61,8 +61,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'myuser',
-        'PASSWORD': 'mypassword',
+        'USER': 'vmsadmin',
+        'PASSWORD': '0xdeadbeef',
         'HOST': 'localhost',
```

Comment:
Isn't this what the person who is installing choose? So instead of myuser and mypassword you need to type your own.